src/%IPM/ResourceProcessor/LegacyLocalizedMessages.cls

```diff
@@ -159,7 +159,7 @@ Method OnResolveChildren(ByRef pResourceArray) As %Status
 			Set tPtr = 0
 			While $ListNext(tIncludeList,tPtr,tInclude) {
 				If $Data(tInclude) || ($Piece(..includeFiles,":") '= "") {
-					Set tName = $Piece(..includeFiles,":")_".INC"
+					Set tName = $Piece(..includeFiles,":",2)_".INC"
```

Comment:
What is this from?

Reply:
Ah this is made in %ZHSLIB.PackageManager.Developer.Processor.LegacyErrorMessages. I did not look into it in depth though
![image](https://user-images.githubusercontent.com/94547358/167305700-cd80963e-252d-45b3-9c4b-22590265d565.png)